`from` breaks when a struct member is feature gated

Generation of `from()` breaks when some struct fields are feature gated but not used.

E.g.:

```
❯ cat -p src/main.rs 
use optfield::optfield;

#[optfield(OptFoo, attrs, field_attrs, from)]
struct Foo {
    a: String,
    #[cfg(feature = "foo")]
    b: String,
}

fn main() {
    println!("Hello, world!");
}
```

... expands to this:

```
#![feature(prelude_import)]                                                    
#[prelude_import]    
use std::prelude::rust_2021::*;
#[macro_use]                        
extern crate std;      
use optfield::optfield;
struct Foo {
    a: String,      
}                                      
struct OptFoo {
    a: Option<String>,                                                                                                                                         
}                                    
impl From<Foo> for OptFoo {                                                                                                                                    
    fn from(item: Foo) -> OptFoo {
        OptFoo {
            a: Some(item.a),                                                   
            b: Some(item.b),
        }     
    }                      
}             
fn main() {
    {
        ::std::io::_print(format_args!("Hello, world!\n"));
    };                        
}
```

... which, unless the feature foo is enabled, fails compiling:

```
❯ cargo build
   Compiling optfield_cfgtest v0.1.0 (/home/kaspar/tmp/optfield_cfgtest)
error[E0560]: struct `OptFoo` has no field named `b`
 --> src/main.rs:7:5
  |
7 |     b: String,
  |     ^ `OptFoo` does not have this field
  |
  = note: all struct fields are already assigned

error[E0609]: no field `b` on type `Foo`
 --> src/main.rs:7:5
  |
7 |     b: String,
  |     ^ unknown field
  |
help: a field with a similar name exists
  |
7 |     a: String,
  |     ~


```

--- a/src/from.rs
+++ b/src/from.rs
@@ -1,5 +1,5 @@
 use proc_macro2::TokenStream;
-use quote::quote;
+use quote::{quote, ToTokens};
 use syn::{Fields, Index, ItemStruct};
 
 use crate::args::Args;
@@ -33,6 +33,14 @@ fn field_bindings(fields: &Fields, args: &Args) -> TokenStream {
     let mut tokens = TokenStream::new();
 
     for (i, field) in fields.iter().enumerate() {
+        let mut cfg_attrs = TokenStream::new();
+
+        for attr in field.attrs.iter() {
+            if attr.path().is_ident("cfg") {
+                attr.to_tokens(&mut cfg_attrs);
+            }
+        }
+
         let field_name = match &field.ident {
             // means that original item is a tuple struct
             None => {
@@ -45,10 +53,12 @@ fn field_bindings(fields: &Fields, args: &Args) -> TokenStream {
 
         let field_tokens = if fields::is_option(field) && !args.rewrap {
             quote! {
+                #cfg_attrs
                 #field_name: item.#field_name,
             }
         } else {
             quote! {
+                #cfg_attrs
                 #field_name: Some(item.#field_name),
             }
         };

